execute/plugin_functions.go

```diff
@@ -391,6 +454,7 @@ func getConsensusObservation(
 	destChainSelector cciptypes.ChainSelector,
 	F int,
 	fChain map[cciptypes.ChainSelector]int,
+	messageExecutionEconomicsEnabled bool,
```

Comment:
I think you could leave this out. The prices would be empty by default on account of no price observations being present. Add a check to VerifiyObservation that there are no price observations when the flag is false.